src/ui/public/agg_types/buckets/terms.js

```diff
@@ -130,6 +132,13 @@ define(function (require) {
 
             let orderAgg = agg.params.orderAgg || vis.aggs.getResponseAggById(agg.params.orderBy);
 
+            const orderBy = orderAgg.type.name;
+            const sortOrder = agg.params.order.val;
```

Comment:
I prefer `sortOrder` over `dir`, but I'm not a fan of two variables tracking the same thing.